locations/spiders/montessori_school.py

```
@@ -1,41 +1,22 @@
+from scrapy.http import Response
+from scrapy.spiders import SitemapSpider
 
 from locations.categories import Categories, apply_category
 from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
 
 
-class MontessoriSchoolSpider(scrapy.Spider):
+class MontessoriSchoolSpider(SitemapSpider, StructuredDataSpider):
     name = "montessori_school"
     item_attributes = {"brand": "Montessori School"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)